source/_docs/continuous-integration.md

```diff
@@ -17,18 +17,69 @@ Continuous Integration (CI) is a method of running automated unit and integratio
-
-After enabling the SimpleTest module, you can use Drush to remotely execute SimpleTest on your Pantheon site. For example, if you want to execute the UserSaveTestCase test and generate XML output into a writeable directory, use the following command:
-
-    SITE_NAME=yoursitename
-    ENV=dev
-    drush @pantheon.$SITE_NAME.$ENV test-run -l https://$ENV-$SITE_NAME.pantheon.io/ UserSaveTestCase --xml='sites/default/files'
-
-The end result is written to https://dev-yoursitename.pantheon.io/sites/default/files/UserSaveTestCase.xml
-
+## Drupal SimpleTest
+
+[SimpleTest](https://drupal.org/project/simpletest){.external} is a testing framework based on the [SimpleTest PHP library](https://github.com/simpletest/simpletest){.external} that is included with Drupal core. If you are creating a custom web application, you should consider including SimpleTests of your module functionality.
+
+[SiteTest](https://www.drupal.org/project/site_test){.external} is a contrib module designed to allow running tests directly against your sites code instead of a base Drupal clone of your site.  This module is recommended for use of SimpleTest on Pantheon.
+
+<div class="alert alert-info" role="alert" markdown="1">
+#### Note {.info}
+The drush test-run command was dropped in Drush 7 and 8. See [this GitHub issue](https://github.com/drush-ops/drush/issues/1362){.external} for more details.
+</div>
+
+To run tests on Pantheon:
+
+1. Enable `site_test`:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- en site_test -y
+   ```
+
+1. Clear the cache immediately before running test to avoid strange failures:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- cc all
+   ```
+
+1. Get the absolute path before you can run the script:
+
+   ```bash
+   terminus drush $SITE_NAME.$ENV_NAME -- eval "echo DRUPAL_ROOT"
+   ```
+
+  In the command above, you may need to strip out warnings by ending the command with `2>/dev/null`.
+
+The full command will look something like this:
+
+```bash
+terminus drush $SITE_NAME.$ENV_NAME -- exec php `terminus drush $SITE_NAME.$ENV_NAME -- eval "echo DRUPAL_ROOT" 2>/dev/null`/scripts/run-tests.sh --url http://$ENV_NAME-$SITE_NAME.pantheonsite.io OptionalTestGroup
+```
+
+In the above command the `--url` option is required to be passed as multidevs do not respond to `localhost`.
+
+A full CircleCI command might look similar to this:
+
+```
+      - run:
+          name: Test simpletest
+          command: |
+            if [ "${CIRCLE_BRANCH}" != "master" ]; then
+
+              terminus drush $SITE_NAME.$ENV_NAME -- en site_test -y
+
+              # If you don't clear the cache immediately before running tests
+              # we get the html gibberish instead of a passing test.
+              terminus drush $SITE_NAME.$ENV_NAME -- cc all
+
+              # NOTE: It is a requirement to use Terminus <=1.9
```

Comment:
@alexfornuto This is my bad.  I should not have listed a specific version of terminus here as the new version required is 2.0.  This should read it is a requirement to use the latest version of terminus to exclude notices about needing to upgrade.  (there may be a terminus option to exclude notices I am not aware of we could put in the command but for now documentation of using the latest version will cover the error)

Reply:
@ccjjmartin do the warnings break the test?

Reply:
@alexfornuto Yes, if the warnings are displayed they will break the test since we have to build a path to the test files using the output of `echo DRUPAL_ROOT` the `[WARNING] Upgrade Terminus` string gets appended to our file path and that is how it breaks.

Reply:
Resolved by https://github.com/pantheon-systems/documentation/pull/4524/commits/d493ff6235c638b187eb11b0c180418b39f8ad1e